devops/ansible/roles/large_image/tasks/mapnik.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install gdal system package
+  apt:
+    name:
+      - gdal-bin
+      - libgdal-dev
+      - libmapnik-dev
+      - python-gdal
```

Comment:
Can you just add a comment explaining why the apt package `python-gdal` is also being installed, given that we install and use `gdal` from pip. Are there are secondary package dependencies or system configurations that the apt package install provides?